src/main/scala/ch/epfl/bluebrain/nexus/admin/persistence/EventSerializer.scala

```diff
@@ -0,0 +1,65 @@
+package ch.epfl.bluebrain.nexus.admin.persistence
+
+import java.nio.charset.StandardCharsets.UTF_8
+
+import akka.actor.ExtendedActorSystem
+import akka.serialization.SerializerWithStringManifest
+import ch.epfl.bluebrain.nexus.admin.config.Settings
+import ch.epfl.bluebrain.nexus.admin.organizations.OrganizationEvent
+import ch.epfl.bluebrain.nexus.admin.projects.ProjectEvent
+import ch.epfl.bluebrain.nexus.iam.client.config.IamClientConfig
+import ch.epfl.bluebrain.nexus.rdf.Iri
+import ch.epfl.bluebrain.nexus.rdf.Iri.AbsoluteIri
+import io.circe.{Decoder, Encoder, Printer}
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.parser.parse
+
+class EventSerializer(system: ExtendedActorSystem) extends SerializerWithStringManifest {
```

Comment:
you don't need to implement the whole `SerializerWithStringManifest `. You can use the `AkkaCoproductSerializer` that we have in `nexus-service` or `nexus-commons` and simplify all this with just
```scala
 class EventSerializer
      extends AkkaCoproductSerializer[
        OrganizationEvent :+: ProjectEvent :+: CNil](1215)
```